### PR TITLE
SERVICE: Created IBranchService Interface And BranchService Class.

### LINKED ISSUE #81: SERVICE: Create IBranchService Interface And BranchService Class.
Create an IBranchService Interface inside the Service Layer and implement the interface in the BranchService class. Also, create a separate folder named BranchServices for this class.

### PR DESCRIPTION
**Pull request by khabibulladev**
Closes #81

### CODE REVIEW

File: BelissimoCloneWPF.Service/DTOs/Branches/BranchForUpdateDTO.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace BelissimoCloneWPF.Service.DTOs.Branches
+{
+    public class BranchForUpdateDTO
+    {
+        public string Name { get; set; }
+        public string Location { get; set; }
+    }
+}
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/DTOs/Branches/BranchForUpdateDTO.cs:8]: remove this class 
use `BranchForCreationDTO` instead

File: BelissimoCloneWPF.Service/DTOs/Branches/BranchForViewDTO.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace BelissimoCloneWPF.Service.DTOs.Branches
+{
+    public class BranchForViewDTO
+    {
+        public string Name { get; set; }
+        public string Location { get; set; }
+    }
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/DTOs/Branches/BranchForViewDTO.cs:7]: Add Id property

File: BelissimoCloneWPF.Service/Services/Branches/BranchService.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+        public ValueTask<BranchForViewDTO> GetAsync(Expression<Func<Branch, bool>> expression)
+        {
+            throw new NotImplementedException();
+        }
+
+        public ValueTask<BranchForViewDTO> UpdateAsync(int id, BranchForUpdateDTO branchForUpdateDTO)
+        {
+            throw new NotImplementedException();
+        }
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/Services/Branches/BranchService.cs:51]: ?

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]

File: BelissimoCloneWPF.Service/Services/Branches/BranchService.cs (hunk 3 above)
Comment (khabibulladev) [reply BelissimoCloneWPF.Service/Services/Branches/BranchService.cs]: Sorry Corrected.

**Pull request closed without merging**